packages/instant-apps-components/src/components/instant-apps-interactive-legend/support/helpers.ts

```diff
@@ -212,8 +212,7 @@ export function handlePredominanceExpression(elementInfo: any, featureLayerView:
       if (elementInfo.value === field) {
         return;
       }
-      const sqlQuery = `(${elementInfo.value} > ${field} OR (${field} IS NULL AND ${elementInfo.value} <> 0 AND ${elementInfo.value} IS NOT NULL))`;
-
+      const sqlQuery = `(${elementInfo.value} > ${field} OR (${field} IS NULL AND ${elementInfo.value} IS NOT NULL AND (${elementInfo.value} > 0 AND ${elementInfo.value} < 0)))`;
```

Comment:
How would this work?  `(${elementInfo.value} > 0 AND ${elementInfo.value} < 0)`

Reply:
Oops you're right, I updated `AND` to `OR`.  `${elementInfo.value} <> 0` wasn't working properly so I was trying to write an alternative to that.